src/lib/cimb_translator/test/CimbReaderTest.cpp

```diff
@@ -52,12 +52,48 @@ TEST_CASE( "CimbReaderTest/testReadOnce", "[unit]" )
 	assertFalse(cr.done());
 }
 
-TEST_CASE( "CimbReaderTest/testSample", "[unit]" )
+TEST_CASE( "CimbReaderTest/testSample.colormode0", "[unit]" )
 {
 	cv::Mat sample = TestCimbar::loadSample("6bit/4color_ecc30_fountain_0.png");
 
 	CimbDecoder decoder(4, 2);
-	CimbReader cr(sample, decoder);
+	CimbReader cr(sample, decoder, 0);
```

Comment:
This is the old test case, before #91 . As long as we're supporting mode 4C, it makes sense to keep a few tests around to make sure it doesn't break.